src/battle_anim_fire.c

```diff
@@ -1296,6 +1296,13 @@ static void AnimWillOWispFire(struct Sprite *sprite)
     sprite->data[3] += 0xC0 * 2;
     sprite->data[4] += 0xA0;
 
+    if(IsDoubleBattle() && !IsContest() &&
+        IsBattlerAlive(BATTLE_PARTNER(gBattleAnimTarget)) &&
+        GetMoveTarget(gAnimMoveIndex) == MOVE_TARGET_BOTH)
+    {
+        SetAverageBattlerPositions(gBattleAnimTarget, TRUE, &sprite->x, &sprite->y);
+    }
```

Comment:
```suggestion
    if (IsDoubleBattle()
     && !IsContest()
     && IsBattlerAlive(BATTLE_PARTNER(gBattleAnimTarget))
     && GetMoveTarget(gAnimMoveIndex) == MOVE_TARGET_BOTH)
        SetAverageBattlerPositions(gBattleAnimTarget, TRUE, &sprite->x, &sprite->y);
```
Alignment and curly brackets